docs/vsphere-golden-image.md

```diff
@@ -144,10 +144,16 @@ To deploy the Windows VM as a reusable image, you have to first generalize the V
 such as installed drivers. Running the `sysprep` command with a *unattend* answer file generalizes the image and 
 makes it ready for future deployments, maintaining all the changes needed for the WMCO installation. 
 
-You can find an example [unattend.xml](unattend.xml) file in this repository. It is just an example and, it **should 
-not** be used directly. You must customize the example to fit your needs.
+A reference [unattend.xml](unattend.xml) file is provided, it **cannot** be used directly. You must customize it
+replacing the `WindowsPassword` parameter with the desired password for the Administrator user. It's recommended to
+follow the [Microsoft best practices for choosing the password.](https://docs.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/password-must-meet-complexity-requirements)
 
-To execute the `sysprep` command use:
+Please note that, the value of the `ComputerName` property in the [unattend.xml](unattend.xml) file must follows the
```

Comment:
Is `property` the correct terminology for XML?

Reply:
Good catch; `element` it is. Still, simplicity will re-frame it as:

"Please note that, the `ComputerName` value in the [unattend.xml](unattend.xml) file must follows the..."